### PR TITLE
Create own auction

### PR DESCRIPTION
**Pull request by razacodespython**

### CODE REVIEW

File: portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx [hunk 1]
```diff
@@ -14,18 +14,18 @@ updated: ""
...
 
 ## Dashboard setup
 
-The first thing we need to do is head over to the [dashboard](https://thirdweb.com/dashboard) and create a project and a **Marketplace** module. if you don't know how to do this, check out [this guide](/learn/introduction).
+The first thing we need to do is head over to the [dashboard](https://thirdweb.com/dashboard) and create a **Marketplace** contract. if you don't know how to do this, check out [this guide](/learn/introduction).
 
-Once you've created the module, it's time to list our first NFT, but we don't need the dashboard for that! I mean we could, but why use GUI if we can use TypeScript!?
+Once you've created the contract, it's time to list our first NFT, but we don't need the dashboard for that! I mean we could, but why use GUI if we can use TypeScript!?
 
 ![](/assets/portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk/17-marketplace.png)
```
Comment (nachoiacovino) [inline portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx:30]: All the screenshots need to be updated with the new nightly dashboard

File: portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx [hunk 2]
```diff
@@ -61,27 +61,27 @@ Click [here](/learn/connect-to-blockchain) to learn how to connect to the blockc
...
 
-Now we need to define which module we want to use. This is the module we just created inside our project Minting NFTs with TypeScript.
-We called the module '**Marketplace written in TypeScript**'.
-This module has an address. We need to pass the address here. You can find the address here, under '**Marketplace written in TypeScript**':
+Now we need to define which contract we want to use. This is the contract we just created inside our dashboard.
+We called the contract '**Marketplace written in TypeScript**'.
+This contract has an address. We need to pass the address here. You can find the address here, under '**Marketplace written in TypeScript**':
 
-![](/assets/portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk/20-module.png)
+![](/assets/portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk/20-contract.png)
```
Comment (nachoiacovino) [inline portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx]: this picture doesn't load in the guide, did you modify the image name?

File: portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx [hunk 3]
```diff
@@ -61,27 +61,27 @@ Click [here](/learn/connect-to-blockchain) to learn how to connect to the blockc
...
+// Initialize market contract by passing in contract address
+const marketPlace = sdk.getMarketplace("0xC06adC34097afa2085324D4192fbE9206059f8e0");
 
 // Declaring the nft smart contract address
 const nftSmartContractAddress = "0xdd25FAEE772FbB1bcB7ba0b2cEE6387A8F82f032"
 
-// Declaring the token module contract if you want to use your own tokens
-const currencySmartContractAddress =
+// Declaring the token contract contract if you want to use your own tokens
+const tokenSmartContractAddress =
```
Comment (nachoiacovino) [inline portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx]:
```suggestion
const tokenAddress =
```

File: portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx [hunk 4]
```diff
@@ -61,27 +61,27 @@ Click [here](/learn/connect-to-blockchain) to learn how to connect to the blockc
...
+In our `index.ts` file we include the following code. First we define a variable `market_smart_contract_address` and store our contract address in here.
 
-Then we create an object that uses the market module inside the SDK with our address.
+Then we create an object that uses the market contract inside the SDK with our address.
 
 ```tsx:index.ts
-// Initialize market module by passing in contract address
-const market = sdk.getMarketModule("0xC06adC34097afa2085324D4192fbE9206059f8e0");
+// Initialize market contract by passing in contract address
+const marketPlace = sdk.getMarketplace("0xC06adC34097afa2085324D4192fbE9206059f8e0");
```
Comment (nachoiacovino) [inline portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx]:
```suggestion
const marketplace = sdk.getMarketplace("0xC06adC34097afa2085324D4192fbE9206059f8e0");
```
let's not capitalize the P in marketplace

File: portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx [hunk 5]
```diff
@@ -95,10 +95,10 @@ We are using the NFT that we created in our previous guide. Each NFT has `token
 const tokenId = "1";
 const tokenIdOffer = 1
 
-market.createAuctionListing({
+marketPlace.createAuctionListing({
```
Comment (nachoiacovino) [inline portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx]:
```suggestion
marketplace.createAuctionListing({
```

File: portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx [hunk 6]
```diff
@@ -95,10 +95,10 @@ We are using the NFT that we created in our previous guide. Each NFT has `token
...
       assetContractAddress: nftSmartContractAddress,
       buyoutPricePerToken: ethers.utils.parseUnits("10"),
-      currencyContractAddress: currencySmartContractAddress,
+      currencyContractAddress: tokenSmartContractAddress,
```
Comment (nachoiacovino) [inline portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx]:
```suggestion
      currencyContractAddress: tokenAddress,
```

File: portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx [hunk 7]
```diff
@@ -122,7 +122,7 @@ Now that you've placed your first auction on your marketplace, someone can make
 Check out the method below, that allows a user to make a bid!
 
 ```tsx:index.ts
-market.makeAuctionListingBid({
+marketPlace.makeAuctionListingBid({
```
Comment (nachoiacovino) [inline portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx]:
```suggestion
marketplace.makeAuctionListingBid({
```

this code is not correct, you need to test the code and make sure it works.

This should be marketplace.auction.makeBid(...

File: portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx (hunk 5 above)
Comment (nachoiacovino) [reply portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx]: also, the code is also not correct, it's marketplace.listing.createListing(...

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
images need updating, also some images are broken and the code is still v1 code.

needs work.

File: portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx (hunk 1 above)
Comment (razacodespython) [reply portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx:30]: done

File: portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx (hunk 2 above)
Comment (razacodespython) [reply portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx]: updated the name

**Pull request merged**